ray-operator/controllers/ray/common/pod.go

```diff
@@ -753,10 +754,16 @@ func generateRayStartCommand(ctx context.Context, nodeType rayv1.RayNodeType, ra
 	if _, ok := rayStartParams["num-gpus"]; !ok {
 		// Scan for resource keys ending with "gpu" like "nvidia.com/gpu".
 		for resourceKey, resource := range resource.Limits {
-			if strings.HasSuffix(string(resourceKey), "gpu") && !resource.IsZero() {
+			resourceKeyString := string(resourceKey)
+			if strings.HasSuffix(resourceKeyString, "gpu") && !resource.IsZero() {
 				rayStartParams["num-gpus"] = strconv.FormatInt(resource.Value(), 10)
 				// For now, only support one GPU type. Break on first match.
 				break
+			} else if resourceKeyString == "aws.amazon.com/neuroncore" && !resource.IsZero() {
```

Comment:
Can you put `aws.amazon.com/neuroncore` into a constant?

Reply:
Made it a constant